composer.json

```diff
@@ -24,13 +24,13 @@
         "predis/predis"  : "v0.8.5",
         "guzzle/guzzle": ">=3.8@stable",
         "doctrine/common": "^2.5",
-        "talis/persona-php-client": ">=3.1"
+        "talis/persona-php-client": "^4.0.0"
```

Comment:
Won't the composer lock file stop the version from changing unless it is updated?